src/components/e-board/BoardMemberThumbnail.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+
+export interface BoardMemberThumbnailProps {
+  boardMemberPhotoUrl: string;
+}
+
+export default function BoardMemberThumbnail(props: BoardMemberThumbnailProps) {
+  return (
+    <div className="border">
+      <Image
+        src={props.boardMemberPhotoUrl}
+        alt={"member"}
+        width={324}
+        height={223}
+        className={"w-[324px] h-[223px]"}
```

Comment:
You gon' set the image height and width twice???

Reply:
Yeahhhhh had to use the next image component because it renders a higher quality image. Weird quirk with it tho it forces me to add a height and width but it made the images different sizes...specifying the height and width in tailwind was the only way to actually get the images to be the same size.